project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ object Dependencies {
   val jacksonV = "2.13.2"
   val jacksonHotfixV = "2.13.2.2" // for when only some of the Jackson libs have hotfix releases
   val nettyV = "4.1.77.Final"
-  val workbenchLibsHash = "2c48b00"
+  val workbenchLibsHash = "0c0401b4-SNAP"
```

Comment:
TODO: update to non-SNAP once https://github.com/broadinstitute/workbench-libs/pull/994 is merged